### PR TITLE
Ability to hide the "branch" column 

### PR DESCRIPTION
**Pull request by notFloran**
#### Description

I cut app.js in 3 files and moved all the configuration stuff in a service.

I added a parameter to indicate if we want to display the "branch" column.
#### Resume
- Bug fix: no
- New feature: yes
- Fixed tickets: #91

### CODE REVIEW

File: app/assets/settings.html [hunk 1]
```diff
@@ -20,6 +20,13 @@
       <input type='text' ng-model="vm.config.refresh_rate" />
     </label>
 
+    <label>Display the "Branch" column
+      <input type='radio' ng-model="vm.config.display_branch_column" ng-value="true" />
+      Yes
+      <input type="radio" ng-model="vm.config.display_branch_column"ng-value="false" >
```
Comment (nclsHart) [inline app/assets/settings.html]: missing space between `ng-model`and `ng-value`

File: app/services.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    }, function() {
+      gitLabManager.logout();
+      deferred.reject('Unauthorized');
+    });
+
+    return deferred.promise;
+  }
+
+  gitLabManager.logout = function() {
+    configManager.clearAll();
```
Comment (nclsHart) [inline app/services.js]: Maybe we should keep the config which is not directly related to GitLab. Refresh rate and display branch column for example.

Comment (notFloran) [inline app/services.js]: @nclsHart What do you think of a **clearCredentialsValues** method instead ?

Comment (nclsHart) [inline app/services.js]: @notFloran seems great.

Comment (notFloran) [inline app/services.js]: @nclsHart It's done !

**Comment by notFloran** [comment]
ping @martin-damien

**Pull request merged**